### PR TITLE
Bug 1252298 - Feature Request: Issue Templates for Components

### PR DESCRIPTION
**Pull request by kyoshino**
Implement component-level bug description (comment) templates that can override product-level templates already implemented in #942.

## Bugzilla link

[Bug 1252298 - Feature Request: Issue Templates for Components](https://bugzilla.mozilla.org/show_bug.cgi?id=1252298)

### CODE REVIEW

File: Bugzilla/Component.pm [hunk 1]
```diff
@@ -374,6 +376,17 @@ sub bug_count {
   return $self->{'bug_count'};
 }
 
+# Lazy-load the bug_description_template column
+sub bug_description_template {
+  my $self = shift;
+  if (!exists $self->{'bug_description_template'}) {
```
Comment (dylanwh) [inline Bugzilla/Component.pm]: If you add a `try { } catch {};` here (and a `use Try::Tiny;` to the top of the module) you could avoid having to do a separate DB code change push

Comment (kyoshino) [reply Bugzilla/Component.pm]: Okay 👍

Comment (kyoshino) [reply Bugzilla/Component.pm]: Looks like `try-catch` isn't required here. Can retrieve 2 templates at once.

**Comment by kyoshino** [comment]
This is still working, ready for review 🙆‍♂

**Comment by kyoshino** [comment]
#1312 is the schema-only PR. I’ll go ahead and merge the other one.

**Comment by kyoshino** [comment]
Adding the **blocking** label until the schema-only PR is deployed.

**Comment by dklawren** [comment]
> 
> 
> #1312 is the schema-only PR. I’ll go ahead and merge the other one.

Ah right. Cool.

**Pull request merged**